packages/gatsby-cli/src/reporter/types.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+export type ActivityTracker = {
+  start(): Function,
+  end(): Function,
+  setStatus(s: string): Function,
+  span: Object,
+}
+
+export type ActivityArgs = {
+  parentSpan?: Object,
+}
+
+export interface Reporter {
```

Comment:
is it possible to mimic these typings? As it's not entirely correct.
https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/index.d.ts#L892-L915

Reply:
Sure! I wasn't aware of the TS typings. I'll sync them.

Reply:
Maybe later these can be generated with flowgen